commands/economy/setpoint.js

```diff
@@ -0,0 +1,116 @@
+const Discord = require("discord.js")
+const { COLOR } = process.env;
+
+module.exports = {
+  name:"setpoint",
+  category:"economy",
+  description:"Set user point add or remove",
+  usage:'<user> <add or remove> <nominal> <reason>',
+  cooldown:1000,
+  run: async(client, message, args) => {
+        let member = message.guild.member(message.mentions.users.first());
+		let author = message.guild.members.cache.get(message.author.id);
+
+		if (message.author.id !== process.env.OWNER) {
+            return message.channel.send("This is owner only command.");
+        }
+
+		if (!member) {
+            return message.channel.send("Please Mention A User");
+        }
+
+        if (!args[1]) {
+            return message.channel.send("You need argument for this command. Argument is `add` or `remove`");
+        }
+
+		if (!args[2]) {
+            return message.channel.send("Please Enter Valid Number");
+        }
+
+		if (args[2] < 1) {
+			return message.channel.send("You Need To Transfer More Than 1");
+        }
+
+		if (isNaN(args[2])) { 
+            return message.channel.send("Thats Not A Number -_-");
+        }
+
+		const reason = args.slice(3).join(" ");
+
+		if (!reason) return message.channel.send("Please enter valid reason");
+
+		let target = await User.findOne({
+			guildID: message.guild.id,
+			userID: member.id,
+		});
+
+		if (author.userID == member.id)
+			return message.channel.send("How Is That Possible");
+		if (member.user.bot) {
+            return message.channel.send("Its A Bot -_-");
+        }
+
+        //args
+        if (args[1].toLowerCase() === "add") {
+            let e = new Discord.MessageEmbed()
+            .setColor(COLOR)
+            .setDescription(
+                `**${message.author.username}** Gives ${member.user.username} ${args[2]} Points\n With reason : ${reason}`,
+            );
+            target.point += Math.floor(parseInt(args[2]));
+            target.save();
+            message.channel.send({ embed: e });
+
+            Guild.findOne({ guildID: message.guild.id }, async (err, data) => {
+                if(!data) {
+                    return message.reply('this guild not have any data');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mhazm/Saino-sama/pullRequest?prid=7538397)
```suggestion
                    return message.reply("this guild not have any data");
```